hexa/pipelines/views.py

```diff
@@ -126,14 +126,15 @@ def run_pipeline(
         return JsonResponse({"error": "Pipeline has no webhook enabled"}, status=400)
 
     # Get the pipeline version
+    pipeline_version = None
     try:
         pipeline_version = pipeline.last_version
         if version_id is not None:
             pipeline_version = PipelineVersion.objects.get(
                 pipeline=pipeline, id=version_id
             )
 
-        if pipeline_version is None:
+        if pipeline.type == PipelineType.ZIPFILE and pipeline_version is None:
```

Comment:
It's weird to let the lines above run if you know that it does not going to work. The logic can be improved.